docs/src/technologies/device_tree_overlays_pt.md

```diff
@@ -0,0 +1,150 @@
+<!--
+    Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Device Tree Overlays for Passthrough
+
+The device tree blob (DTB) is a data structure that describes the hardware 
```

Comment:
Again feels like most minor nitpick of all times, but this file has multiple lines with trailing whitespace. Clearly visible with red color in `git diff` from terminal. Could you please remove those?